plugins/woocommerce/tests/e2e-pw/utils/themes.js

```diff
@@ -1,18 +1,61 @@
 const { exec } = require( 'node:child_process' );
+const { encodeCredentials } = require( '../utils/plugin-utils' );
+const https = require( 'https' );
+const { admin } = require( '../test-data/data' );
+const { BASE_URL } = process.env;
 
 export const DEFAULT_THEME = 'twentytwentythree';
 
 export const activateTheme = ( themeName ) => {
 	return new Promise( ( resolve, reject ) => {
-		const command = `wp-env run tests-cli wp theme install ${ themeName } --activate`;
+		const isLocalhost = BASE_URL.includes( 'localhost' );
+		if ( isLocalhost ) {
```

Comment:
I don't see a good reason for this conditional. Why not using the same api for all environments and drop the cli?

Reply:
Ah, nice. I'm just getting fatal error when I try enabling that plugin on both Pressable and WPCOM sites. `Plugin could not be activated because it triggered a fatal error.`

Reply:
Added theme activation through `test-helper-apis.php` instead.

Reply:
> Added theme activation through `test-helper-apis.php` instead.

Now my initial comment makes sense. We don't need a conditional here. Just use the new API route for all sites.